fixtures/varytec/giga-bar-hex-3.json

```diff
@@ -164,9 +172,190 @@
           "type": "NoFunction"
         },
         {
-          "dmxRange": [15, 255],
+          "dmxRange": [15, 22],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [23, 30],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [31, 40],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [40, 47],
+          "type": "ColorPreset",
+          "comment": "Pure White",
+          "colors": ["#ffffff"]
+        },
```

Comment:
DMX value 40 is ambiguous; which of these two capabilities should it activate?

Reply:
Oops, I should have checked this!